modules/nf-core/circexplorer2/parse/main.nf

```diff
@@ -0,0 +1,46 @@
+process CIRCEXPLORER2_PARSE {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::circexplorer2=2.3.8" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/circexplorer2:2.3.8--pyh864c0ab_1':
+        'quay.io/biocontainers/circexplorer2:2.3.8--pyh864c0ab_1' }"
+
+    input:
+    tuple val(meta), path(fusions)
+
+    output:
+    tuple val(meta), path("*.bed"), emit: junction
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def aligner = "${fusions}".endsWith(".junction") ? "-t STAR" : "${fusions}".endsWith(".txt") ? "-t MapSplice" : "${fusions}".endsWith(".bam") ? "-t BWA" : "-t segemehl"
+    """
+    CIRCexplorer2 \\
+        parse \\
+        $aligner \\
+        $fusions \\
+        -b ${prefix}.junction.bed
```

Comment:
Should just be `.bed`, if there is  risk of a clash, add a warning [^1]

[^1]: https://nf-co.re/docs/contributing/modules#naming-conventions